scripts/seedDB.js

```diff
@@ -3,7 +3,7 @@ const mongoose = require("mongoose");
 const db = require("../models");
 mongoose.Promise = global.Promise;
 
-// This file empties the Books collection and inserts the books below
+// This file empties the Books collection and inserts the books below <--Ha books..
```

Comment:
Haha, yeah, that's a copy and paste error. We could either delete the note or correct it.